src/main/java/net/pms/network/webinterfaceserver/ServerSentEvents.java

```diff
@@ -0,0 +1,107 @@
+package net.pms.network.webinterfaceserver;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.sql.Timestamp;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ServerSentEvents {
+	private static final Logger LOGGER = LoggerFactory.getLogger(ServerSentEvents.class);
+
+	private final Object osLock = new Object();
+	private Thread pingThread;
+
+	public ServerSentEvents() {
+	}
+
+	public ServerSentEvents(OutputStream os) {
+		addEventStream(os);
+	}
+
+	OutputStream os;
+	public final void addEventStream(OutputStream os) {
+		//clean current OutputStream in case of....
```

Comment:
Is this comment incomplete?

Reply:
It's a reminder note for me.
In the previous implementation, pull allow only one control connection (it's normal).
If you open 2 tabs on your explorer, then 2 connections call pull, the first that call will have the data, the other call will be empty.
I'll need to manage that to allow only 1 WebRender by explorer, at least for sse.
For now, it work like before.

Reply:
I added a message when multiple tabs browse the interface to let user now it.
fixed some html issues that was present.